atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/ImmutableTimestampLockManager.java

```diff
@@ -36,9 +36,13 @@ public ImmutableTimestampLockManager(
         this.lockValidityChecker = lockValidityChecker;
     }
 
-    public Optional<ExpiredLocks> getExpiredImmutableTimestampAndCommitLocks(Optional<LockToken> commitLocksToken) {
+    public Optional<ExpiredLocks> getExpiredImmutableTimestampAndCommitLocks(Set<LockToken> commitLocksToken) {
         Set<LockToken> toRefresh = new HashSet<>();
-        commitLocksToken.ifPresent(toRefresh::add);
+
+        // TODO(jakubk): Handle this upstream
+        if (commitLocksToken != null) {
```

Comment:
I've copied this from my previous PR, I bet I've ran into a place where we push nulls to here...